### PR TITLE
Update script.ts file

### PR DESCRIPTION
**Pull request by ferolinojasmin**
Please review my code, thankyou!

### CODE REVIEW

File: script.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    screen_size: "68",
+    price: 40000,
+    print: function (): void {
+        console.log(`I wanna buy a ${this.brand}, since it is
+        a know local brand.
+        I want it to be ${this.color} color, since I like a cool and aesthetic look.
+        I also want it to be ${this.screen_size} inches since I wanna see and experience the whole thing.
+        My budget is around ${this.price}. `);
+    },
+};
```
Comment (Pasionpreciouskaye) [inline script.ts:71]: put the print function

Comment (ferolinojasmin) [reply script.ts:71]: I already edit my code, thanks!

**Review by Pasionpreciouskaye — APPROVED** [review]
Alright! I think your code id okey, You can merge your code now

**Review by IlaganJireh — APPROVED** [review]
Please double check your conflicts before merging, thanks.

**Pull request merged**